website/docs/unsignedchar.md

```diff
@@ -1,12 +1,17 @@
-unsignedchar - This page was auto-generated. Feel free to help us improve the documentation by creating a pull request.
+unsignedchar - Force sign of `char` (note that `char` is still a distinct type from `signed char` and `unsigned char`).
 
 ```lua
 unsignedchar (value)
 ```
 
 ### Parameters ###
 
-`value` - needs documentation.
+`value` is one of:
+
+| Action      | Description                           |
+|-------------|---------------------------------------|
+| Off         | make `char` signed. (default on msc)  |
+| On          | make `char` unsigned.                 |
```

Comment:
I'm pretty sure we've been preferring lists over tables as it doesn't force documenting things that don't need documenting. This API is pretty self-documenting, so the above should only need to be:
> - Off: default on msc
> - On

For examples of where we're using lists over tables, see [`buildstlmodules`](https://premake.github.io/docs/buildstlmodules), [`compileas`](https://premake.github.io/docs/compileas), or [`cppdialect`](https://premake.github.io/docs/cppdialect). Some APIs haven't been updated yet and are still using tables from the wiki days.

Reply:
Changed.